docs/api/python/symbol.md

```diff
@@ -392,6 +392,21 @@ Composite multiple symbols into a new one by an operator.
     argmin
 ```
 
+### Linear Algebra
+
+```eval_rst
+.. autosummary::
+    :nosignatures:
+
+    la_gemm
```

Comment:
no preference on my side concerning namespace. we can choose what we like. 
What is your concern about exposing as operators. That's exactly what people need in order to build loss functions based on advanced linear algebra. Or are you referring just to gemm?